teemaderegister-react/src/actions/CurriculumActions.js

```diff
@@ -31,19 +31,19 @@ export const getCurriculums = () => dispatch => {
       type: types.CURRICULUMS_LOADED,
       curriculums: data.curriculums
     }))
-    .catch(err => {
-      console.log(err)
-    })
+      .catch(err => {
+          console.log(err)
+      })
```

Comment:
please review all unneeded changes that do not go with linter rules in all files, please do it backend and frontend containers. Those UNSAFE_  can be ignored.
```
npm run lint
```

Reply:
parandatud